src/usersAPI/users-controller.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+// Imports:
+const { getUsersFromDB, writeUsersToDB } = require('./users-service');
+
+// Functions names below describes well what does it do :)
```

Comment:
```suggestion
```
It needs to be so by default. No need to comment anything, if it's made in clear and understandable way. Don't overload your code with unnecessary comments - it's hard to read and adds 0% of quality to your code.